Assignment/test.c

```diff
@@ -10,38 +10,33 @@
 #include <string.h>
 #include "conversion.h"
 
-// ANSII color code for green and red text
+// ANSI color code for green and red text
 #define GREEN_TEXT "\033[0;32m"
 #define RED_TEXT "\033[0;31"
 #define RESET_TEXT "\033[0m"
 
-void PrintRightAligned (const char* result, int terminalWidth) {
-   int padding = terminalWidth - (int)strlen (result) - 1;  
-   for (int i = 0; i < padding; i++) {
-      printf (" ");  
-   }
-   printf ("%s\n", result);  
+void PrintResult (const char* result, int terminalWidth) {
+   int padding = terminalWidth - (int)strlen (result) - 1;

```

Comment:
The terminal width is same for both DecToBinaryTest and DecToHexaTest. So it can directly used inside the PrintResult function instead of passing it.